doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -185,6 +185,14 @@ Documentation
 
 In addition to documentation related to the changes listed above, the following documentation has been updated:
 
-* |no_changes_yet_note|
+* Reorganized the contents of the :ref:´ug_app_dev` section:
+
+  * Reduced the toc levels of the subpages.
```

Comment:
```suggestion
  * Reduced the ToC levels of the subpages.
```
Also in line 196.
But to be honest, in my opinion this change could be omitted, as it is on the verge of minor/trivial.